src/test/kotlin/Strategy/strategies/IVAStrategyTest.kt

```diff
@@ -0,0 +1,22 @@
+import Strategy.strategies.*
+import org.hamcrest.CoreMatchers
+import org.junit.Assert.*
+import org.junit.Test
+
+class IVAStrategyTest{
+
+  @Test
+  fun `check that general iva is applied correctly`(){
+    assertThat(GeneralStrategy(2.0), CoreMatchers.`is`(2+2*0.21))
```

Comment:
Shouln't the result be hardcoded? I mean, the strategy is making that same operation. It won't fail never. We should write the exact result for (2 + 2 * 0.21) for seeing if the calculus is returning what we expect. Am I right?